CONTRIBUTING.md

```diff
@@ -178,7 +178,9 @@ language is used within pull-request comments:
 
   - ACK means "I have tested the code and I agree it should be merged";
   - NACK means "I disagree this should be merged", and must be accompanied by
-    sound technical justification. NACKs without accompanying reasoning may be disregarded;
+    sound technical justification (or in certain cases of copyright/patent
```

Comment:
add `/licensing` because patches may also be unable to be included if the licensing is incompatible with the project.